packages/hint/src/lib/utils/misc/pretty-print-array.ts

```diff
@@ -0,0 +1,33 @@
+/*
+ * Returns an array pretty printed.
+ *
+ * e.g.:
+ *
+ *  []          =>
+ *  [1]         => '1'
+ *  [1, 2]      => '1' and '2'
+ *  [1, '2', 3]   => '1', '2', and '3'
+ *
+ */
+
+export default (array: (string|number)[]): string => {
```

Comment:
Use the format `Array<string | number>` for consistency.

Reply:
Fixed.